spec/datastreams/rights_metadata_spec.rb

```diff
@@ -240,6 +240,32 @@
         </rightsMetadata>
       XML
     end
+    let(:cdl_no_download_rights_xml) do
+      <<-XML
+        <rightsMetadata>
+          <copyright>
+            <human type="copyright">Courtesy of the Revs Institute for Automotive Research. All rights reserved unless otherwise indicated.</human>
+          </copyright>
+          <access type="discover">
+            <machine>
+              <world/>
+            </machine>
+          </access>
+          <access type="read">
+            <machine>
+              <cdl>
```

Comment:
Shouln't `<cdl />`  be separate from and in addition to a `<group rule="no-download">stanford</group>`. that way when `<cdl />` is removed at the end of the term, we just remove the one node?

Reply:
the way the rights were specified by chris beer here https://docs.google.com/document/d/14JPhnBefJWcKZ0uFePZ1yn555aZwBoBigrE9-bsLbmg under the "straw proposal" section, he had the no-download node within the <cdl> node, like this:

```
<cdl>
  <group rule="no-download">stanford</group>
</cdl>
```

I'm not sure if that is important to drive the behavior.  But I believe removing the <cdl> node at the end will take that embedded node with it, right?

Reply:
Yes and since that would take that away, don't we want something to fall back to?

Reply:
Also it would be great if this aligned with the description in https://github.com/sul-dlss/argo/issues/2200

Reply:
Good call, I fixed the ticket description.   We may need to run this by Chris again to be sure we understand the intent.  I think the idea is that when this block is taken away, a new read block would be written to make it citation only again.

Reply:
Checked with Chris and the ticket is accurate.